workflow-templates/check-license.yml

```
@@ -59,13 +59,29 @@ jobs:
 
           echo "result=$RESULT" >> $GITHUB_OUTPUT
 
-  check-license:
+  build:
```

```suggestion
  check-license:
```
The original job key name is more appropriate.